src/resources/clientScopes.ts

```diff
@@ -0,0 +1,335 @@
+import ClientScopeRepresentation from '../defs/clientScopeRepresentation';
+import Resource from './resource';
+import {KeycloakAdminClient} from '../client';
+import ProtocolMapperRepresentation from '../defs/protocolMapperRepresentation';
+import MappingsRepresentation from '../defs/mappingsRepresentation';
+import RoleRepresentation from '../defs/roleRepresentation';
+
+export class ClientScopes extends Resource<{realm?: string}> {
+  public find = this.makeRequest<{}, ClientScopeRepresentation[]>({
+    method: 'GET',
+    path: '/client-scopes',
+  });
+
+  public create = this.makeRequest<ClientScopeRepresentation, void>({
+    method: 'POST',
+    path: '/client-scopes',
+  });
+
+  /**
+   * Client-Scopes by id
+   */
+
+  public findOneById = this.makeRequest<
+    {id: string},
+    ClientScopeRepresentation
+  >({
+    method: 'GET',
+    path: '/client-scopes/{id}',
+    urlParamKeys: ['id'],
+    catchNotFound: true,
+  });
+
+  public updateById = this.makeUpdateRequest<
+    {id: string},
+    ClientScopeRepresentation,
+    void
+  >({
+    method: 'PUT',
+    path: '/client-scopes/{id}',
+    urlParamKeys: ['id'],
+  });
+
+  public delById = this.makeRequest<{id: string}, void>({
+    method: 'DELETE',
+    path: '/client-scopes/{id}',
+    urlParamKeys: ['id'],
+  });
```

Comment:
Unlike roles, client scopes you can only access by id, so I think these functions should match their equivalents in other resources, such as clients, which don't have `ById` in

Reply:
Good catch. You mean - rename it to just `del`. Right?

Will change it! Thanks.